FIAT/enriched.py

```diff
@@ -66,24 +67,37 @@ def __init__(self, A, B):
         # the entities of the constituent elements
         Adofs = A.entity_dofs()
         Bdofs = B.entity_dofs()
+
         offset = A.space_dimension()  # number of entities belonging to A
-        entity_ids = {}
+        self.entity_ids = {}
 
         for ent_dim in Adofs:
-            entity_ids[ent_dim] = {}
+
+            self.entity_ids[ent_dim] = {}
+
             for ent_dim_index in Adofs[ent_dim]:
                 entlist = copy(Adofs[ent_dim][ent_dim_index])
                 entlist += [c + offset for c in Bdofs[ent_dim][ent_dim_index]]
-                entity_ids[ent_dim][ent_dim_index] = entlist
+                self.entity_ids[ent_dim][ent_dim_index] = entlist
 
         # set up dual basis - just concatenation
         nodes = A.dual_basis() + B.dual_basis()
-        self.dual = dual_set.DualSet(nodes, self.ref_el, entity_ids)
+        self.dual = dual_set.DualSet(nodes, self.ref_el, self.entity_ids)
 
     def degree(self):
         """Return the degree of the (embedding) polynomial space."""
         return self.polydegree
 
+#    def entity_dofs(self):
```

Comment:
No commented out code please. For storing history and non-functional ideas we have the version control system.